stor/exceptions.py

```diff
@@ -83,6 +83,13 @@ class ConflictError(RemoteError):
     """
 
 
+class TargetExistsError(RemoteError):
+    """Thrown when a destination target already exists on DX for a file or folder
+    that is being uploaded/copied/moved, etc.
```

Comment:
I thought we just deleted the target path?

My understanding was that `TargetExistsError` gets thrown when we can't construct a valid path on copying directories?

Reply:
Updated the doc to say it is thrown only while copying folders. Yes, files are deleted during clashes.

Reply:
do docs note that files are deleted during clashes?

Reply:
Yes docs/docstrings elsewhere note that. That does not need to be here.